2024-08-14 05:22:21,507 INFO django.utils.autoreload Watching for file changes with StatReloader
2024-08-14 06:38:17,465 INFO django.utils.autoreload Watching for file changes with StatReloader
2024-08-14 06:38:35,778 WARNING django.request Not Found: /
2024-08-14 06:38:35,778 WARNING django.server "GET / HTTP/1.1" 404 3042
2024-08-14 06:38:52,095 INFO django.server "GET /admin/ HTTP/1.1" 302 0
2024-08-14 06:38:52,118 INFO django.server "GET /admin/login/?next=/admin/ HTTP/1.1" 200 4181
2024-08-14 06:38:52,141 INFO django.server "GET /static/admin/css/dark_mode.css HTTP/1.1" 200 2929
2024-08-14 06:38:52,141 INFO django.server "GET /static/admin/css/base.css HTTP/1.1" 200 21310
2024-08-14 06:38:52,142 INFO django.server "GET /static/admin/css/login.css HTTP/1.1" 200 958
2024-08-14 06:38:52,142 INFO django.server "GET /static/admin/css/nav_sidebar.css HTTP/1.1" 200 2694
2024-08-14 06:38:52,143 INFO django.server "GET /static/admin/css/responsive.css HTTP/1.1" 200 18559
2024-08-14 06:38:52,143 INFO django.server "GET /static/admin/js/theme.js HTTP/1.1" 200 1943
2024-08-14 06:38:52,144 INFO django.server "GET /static/admin/js/nav_sidebar.js HTTP/1.1" 200 3063
2024-08-14 06:39:09,154 ERROR django.request Internal Server Error: /admin/login/
Traceback (most recent call last):
  File "/Users/vitalypateshman/Study/Diplom/.venv/lib/python3.9/site-packages/django/db/backends/utils.py", line 89, in _execute
    return self.cursor.execute(sql, params)
psycopg2.errors.UndefinedTable: relation "accounts_customuser" does not exist
LINE 1: ...is_active", "accounts_customuser"."is_staff" FROM "accounts_...
                                                             ^


The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/Users/vitalypateshman/Study/Diplom/.venv/lib/python3.9/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/Users/vitalypateshman/Study/Diplom/.venv/lib/python3.9/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Users/vitalypateshman/Study/Diplom/.venv/lib/python3.9/site-packages/django/utils/decorators.py", line 46, in _wrapper
    return bound_method(*args, **kwargs)
  File "/Users/vitalypateshman/Study/Diplom/.venv/lib/python3.9/site-packages/django/views/decorators/cache.py", line 62, in _wrapper_view_func
    response = view_func(request, *args, **kwargs)
  File "/Users/vitalypateshman/Study/Diplom/.venv/lib/python3.9/site-packages/django/contrib/admin/sites.py", line 441, in login
    return LoginView.as_view(**defaults)(request)
  File "/Users/vitalypateshman/Study/Diplom/.venv/lib/python3.9/site-packages/django/views/generic/base.py", line 104, in view
    return self.dispatch(request, *args, **kwargs)
  File "/Users/vitalypateshman/Study/Diplom/.venv/lib/python3.9/site-packages/django/utils/decorators.py", line 46, in _wrapper
    return bound_method(*args, **kwargs)
  File "/Users/vitalypateshman/Study/Diplom/.venv/lib/python3.9/site-packages/django/views/decorators/debug.py", line 92, in sensitive_post_parameters_wrapper
    return view(request, *args, **kwargs)
  File "/Users/vitalypateshman/Study/Diplom/.venv/lib/python3.9/site-packages/django/utils/decorators.py", line 46, in _wrapper
    return bound_method(*args, **kwargs)
  File "/Users/vitalypateshman/Study/Diplom/.venv/lib/python3.9/site-packages/django/utils/decorators.py", line 134, in _wrapper_view
    response = view_func(request, *args, **kwargs)
  File "/Users/vitalypateshman/Study/Diplom/.venv/lib/python3.9/site-packages/django/utils/decorators.py", line 46, in _wrapper
    return bound_method(*args, **kwargs)
  File "/Users/vitalypateshman/Study/Diplom/.venv/lib/python3.9/site-packages/django/views/decorators/cache.py", line 62, in _wrapper_view_func
    response = view_func(request, *args, **kwargs)
  File "/Users/vitalypateshman/Study/Diplom/.venv/lib/python3.9/site-packages/django/contrib/auth/views.py", line 90, in dispatch
    return super().dispatch(request, *args, **kwargs)
  File "/Users/vitalypateshman/Study/Diplom/.venv/lib/python3.9/site-packages/django/views/generic/base.py", line 143, in dispatch
    return handler(request, *args, **kwargs)
  File "/Users/vitalypateshman/Study/Diplom/.venv/lib/python3.9/site-packages/django/views/generic/edit.py", line 152, in post
    if form.is_valid():
  File "/Users/vitalypateshman/Study/Diplom/.venv/lib/python3.9/site-packages/django/forms/forms.py", line 201, in is_valid
    return self.is_bound and not self.errors
  File "/Users/vitalypateshman/Study/Diplom/.venv/lib/python3.9/site-packages/django/forms/forms.py", line 196, in errors
    self.full_clean()
  File "/Users/vitalypateshman/Study/Diplom/.venv/lib/python3.9/site-packages/django/forms/forms.py", line 434, in full_clean
    self._clean_form()
  File "/Users/vitalypateshman/Study/Diplom/.venv/lib/python3.9/site-packages/django/forms/forms.py", line 455, in _clean_form
    cleaned_data = self.clean()
  File "/Users/vitalypateshman/Study/Diplom/.venv/lib/python3.9/site-packages/django/contrib/auth/forms.py", line 250, in clean
    self.user_cache = authenticate(
  File "/Users/vitalypateshman/Study/Diplom/.venv/lib/python3.9/site-packages/django/views/decorators/debug.py", line 42, in sensitive_variables_wrapper
    return func(*func_args, **func_kwargs)
  File "/Users/vitalypateshman/Study/Diplom/.venv/lib/python3.9/site-packages/django/contrib/auth/__init__.py", line 77, in authenticate
    user = backend.authenticate(request, **credentials)
  File "/Users/vitalypateshman/Study/Diplom/.venv/lib/python3.9/site-packages/django/contrib/auth/backends.py", line 46, in authenticate
    user = UserModel._default_manager.get_by_natural_key(username)
  File "/Users/vitalypateshman/Study/Diplom/.venv/lib/python3.9/site-packages/django/contrib/auth/base_user.py", line 54, in get_by_natural_key
    return self.get(**{self.model.USERNAME_FIELD: username})
  File "/Users/vitalypateshman/Study/Diplom/.venv/lib/python3.9/site-packages/django/db/models/manager.py", line 87, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
  File "/Users/vitalypateshman/Study/Diplom/.venv/lib/python3.9/site-packages/django/db/models/query.py", line 633, in get
    num = len(clone)
  File "/Users/vitalypateshman/Study/Diplom/.venv/lib/python3.9/site-packages/django/db/models/query.py", line 380, in __len__
    self._fetch_all()
  File "/Users/vitalypateshman/Study/Diplom/.venv/lib/python3.9/site-packages/django/db/models/query.py", line 1881, in _fetch_all
    self._result_cache = list(self._iterable_class(self))
  File "/Users/vitalypateshman/Study/Diplom/.venv/lib/python3.9/site-packages/django/db/models/query.py", line 91, in __iter__
    results = compiler.execute_sql(
  File "/Users/vitalypateshman/Study/Diplom/.venv/lib/python3.9/site-packages/django/db/models/sql/compiler.py", line 1562, in execute_sql
    cursor.execute(sql, params)
  File "/Users/vitalypateshman/Study/Diplom/.venv/lib/python3.9/site-packages/django/db/backends/utils.py", line 102, in execute
    return super().execute(sql, params)
  File "/Users/vitalypateshman/Study/Diplom/.venv/lib/python3.9/site-packages/django/db/backends/utils.py", line 67, in execute
    return self._execute_with_wrappers(
  File "/Users/vitalypateshman/Study/Diplom/.venv/lib/python3.9/site-packages/django/db/backends/utils.py", line 80, in _execute_with_wrappers
    return executor(sql, params, many, context)
  File "/Users/vitalypateshman/Study/Diplom/.venv/lib/python3.9/site-packages/django/db/backends/utils.py", line 89, in _execute
    return self.cursor.execute(sql, params)
  File "/Users/vitalypateshman/Study/Diplom/.venv/lib/python3.9/site-packages/django/db/utils.py", line 91, in __exit__
    raise dj_exc_value.with_traceback(traceback) from exc_value
  File "/Users/vitalypateshman/Study/Diplom/.venv/lib/python3.9/site-packages/django/db/backends/utils.py", line 89, in _execute
    return self.cursor.execute(sql, params)
django.db.utils.ProgrammingError: relation "accounts_customuser" does not exist
LINE 1: ...is_active", "accounts_customuser"."is_staff" FROM "accounts_...
                                                             ^

2024-08-14 06:39:09,165 ERROR django.server "POST /admin/login/?next=/admin/ HTTP/1.1" 500 224495
2024-08-14 06:39:26,670 INFO django.server "GET /admin/login/?next=/admin/ HTTP/1.1" 200 4181
2024-08-14 06:39:40,550 WARNING django.request Method Not Allowed: /register/
2024-08-14 06:39:40,550 WARNING django.server "GET /register/ HTTP/1.1" 405 5631
2024-08-14 06:39:40,567 INFO django.server "GET /static/rest_framework/js/ajax-form.js HTTP/1.1" 200 3796
2024-08-14 06:39:40,569 INFO django.server "GET /static/rest_framework/css/prettify.css HTTP/1.1" 200 817
2024-08-14 06:39:40,569 INFO django.server "GET /static/rest_framework/css/default.css HTTP/1.1" 200 1152
2024-08-14 06:39:40,570 INFO django.server "GET /static/rest_framework/css/bootstrap-tweaks.css HTTP/1.1" 200 3426
2024-08-14 06:39:40,573 INFO django.server "GET /static/rest_framework/js/csrf.js HTTP/1.1" 200 1793
2024-08-14 06:39:40,576 INFO django.server "GET /static/rest_framework/js/default.js HTTP/1.1" 200 1268
2024-08-14 06:39:40,576 INFO django.server "GET /static/rest_framework/js/prettify-min.js HTTP/1.1" 200 13632
2024-08-14 06:39:40,577 INFO django.server "GET /static/rest_framework/js/load-ajax-form.js HTTP/1.1" 200 59
2024-08-14 06:39:40,577 INFO django.server "GET /static/rest_framework/js/jquery-3.7.1.min.js HTTP/1.1" 200 87533
2024-08-14 06:39:40,578 INFO django.server "GET /static/rest_framework/js/bootstrap.min.js HTTP/1.1" 200 39680
2024-08-14 06:39:40,579 INFO django.server "GET /static/rest_framework/css/bootstrap.min.css HTTP/1.1" 200 121457
2024-08-14 06:39:40,586 INFO django.server "GET /static/rest_framework/img/grid.png HTTP/1.1" 200 1458
2024-08-14 06:58:04,854 INFO django.utils.autoreload Watching for file changes with StatReloader
2024-08-14 06:58:20,614 WARNING django.request Not Found: /
2024-08-14 06:58:20,615 WARNING django.server "GET / HTTP/1.1" 404 3042
2024-08-14 06:58:38,831 INFO django.server "GET /api/ HTTP/1.1" 200 5446
2024-08-14 06:58:52,149 INFO django.server "GET /admin/ HTTP/1.1" 302 0
2024-08-14 06:58:52,172 INFO django.server "GET /admin/login/?next=/admin/ HTTP/1.1" 200 4181
2024-08-14 06:59:00,760 ERROR django.request Internal Server Error: /admin/login/
Traceback (most recent call last):
  File "/Users/vitalypateshman/Study/Diplom/.venv/lib/python3.9/site-packages/django/db/backends/utils.py", line 89, in _execute
    return self.cursor.execute(sql, params)
psycopg2.errors.UndefinedTable: relation "accounts_customuser" does not exist
LINE 1: ...is_active", "accounts_customuser"."is_staff" FROM "accounts_...
                                                             ^


The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/Users/vitalypateshman/Study/Diplom/.venv/lib/python3.9/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/Users/vitalypateshman/Study/Diplom/.venv/lib/python3.9/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Users/vitalypateshman/Study/Diplom/.venv/lib/python3.9/site-packages/django/utils/decorators.py", line 46, in _wrapper
    return bound_method(*args, **kwargs)
  File "/Users/vitalypateshman/Study/Diplom/.venv/lib/python3.9/site-packages/django/views/decorators/cache.py", line 62, in _wrapper_view_func
    response = view_func(request, *args, **kwargs)
  File "/Users/vitalypateshman/Study/Diplom/.venv/lib/python3.9/site-packages/django/contrib/admin/sites.py", line 441, in login
    return LoginView.as_view(**defaults)(request)
  File "/Users/vitalypateshman/Study/Diplom/.venv/lib/python3.9/site-packages/django/views/generic/base.py", line 104, in view
    return self.dispatch(request, *args, **kwargs)
  File "/Users/vitalypateshman/Study/Diplom/.venv/lib/python3.9/site-packages/django/utils/decorators.py", line 46, in _wrapper
    return bound_method(*args, **kwargs)
  File "/Users/vitalypateshman/Study/Diplom/.venv/lib/python3.9/site-packages/django/views/decorators/debug.py", line 92, in sensitive_post_parameters_wrapper
    return view(request, *args, **kwargs)
  File "/Users/vitalypateshman/Study/Diplom/.venv/lib/python3.9/site-packages/django/utils/decorators.py", line 46, in _wrapper
    return bound_method(*args, **kwargs)
  File "/Users/vitalypateshman/Study/Diplom/.venv/lib/python3.9/site-packages/django/utils/decorators.py", line 134, in _wrapper_view
    response = view_func(request, *args, **kwargs)
  File "/Users/vitalypateshman/Study/Diplom/.venv/lib/python3.9/site-packages/django/utils/decorators.py", line 46, in _wrapper
    return bound_method(*args, **kwargs)
  File "/Users/vitalypateshman/Study/Diplom/.venv/lib/python3.9/site-packages/django/views/decorators/cache.py", line 62, in _wrapper_view_func
    response = view_func(request, *args, **kwargs)
  File "/Users/vitalypateshman/Study/Diplom/.venv/lib/python3.9/site-packages/django/contrib/auth/views.py", line 90, in dispatch
    return super().dispatch(request, *args, **kwargs)
  File "/Users/vitalypateshman/Study/Diplom/.venv/lib/python3.9/site-packages/django/views/generic/base.py", line 143, in dispatch
    return handler(request, *args, **kwargs)
  File "/Users/vitalypateshman/Study/Diplom/.venv/lib/python3.9/site-packages/django/views/generic/edit.py", line 152, in post
    if form.is_valid():
  File "/Users/vitalypateshman/Study/Diplom/.venv/lib/python3.9/site-packages/django/forms/forms.py", line 201, in is_valid
    return self.is_bound and not self.errors
  File "/Users/vitalypateshman/Study/Diplom/.venv/lib/python3.9/site-packages/django/forms/forms.py", line 196, in errors
    self.full_clean()
  File "/Users/vitalypateshman/Study/Diplom/.venv/lib/python3.9/site-packages/django/forms/forms.py", line 434, in full_clean
    self._clean_form()
  File "/Users/vitalypateshman/Study/Diplom/.venv/lib/python3.9/site-packages/django/forms/forms.py", line 455, in _clean_form
    cleaned_data = self.clean()
  File "/Users/vitalypateshman/Study/Diplom/.venv/lib/python3.9/site-packages/django/contrib/auth/forms.py", line 250, in clean
    self.user_cache = authenticate(
  File "/Users/vitalypateshman/Study/Diplom/.venv/lib/python3.9/site-packages/django/views/decorators/debug.py", line 42, in sensitive_variables_wrapper
    return func(*func_args, **func_kwargs)
  File "/Users/vitalypateshman/Study/Diplom/.venv/lib/python3.9/site-packages/django/contrib/auth/__init__.py", line 77, in authenticate
    user = backend.authenticate(request, **credentials)
  File "/Users/vitalypateshman/Study/Diplom/.venv/lib/python3.9/site-packages/django/contrib/auth/backends.py", line 46, in authenticate
    user = UserModel._default_manager.get_by_natural_key(username)
  File "/Users/vitalypateshman/Study/Diplom/.venv/lib/python3.9/site-packages/django/contrib/auth/base_user.py", line 54, in get_by_natural_key
    return self.get(**{self.model.USERNAME_FIELD: username})
  File "/Users/vitalypateshman/Study/Diplom/.venv/lib/python3.9/site-packages/django/db/models/manager.py", line 87, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
  File "/Users/vitalypateshman/Study/Diplom/.venv/lib/python3.9/site-packages/django/db/models/query.py", line 633, in get
    num = len(clone)
  File "/Users/vitalypateshman/Study/Diplom/.venv/lib/python3.9/site-packages/django/db/models/query.py", line 380, in __len__
    self._fetch_all()
  File "/Users/vitalypateshman/Study/Diplom/.venv/lib/python3.9/site-packages/django/db/models/query.py", line 1881, in _fetch_all
    self._result_cache = list(self._iterable_class(self))
  File "/Users/vitalypateshman/Study/Diplom/.venv/lib/python3.9/site-packages/django/db/models/query.py", line 91, in __iter__
    results = compiler.execute_sql(
  File "/Users/vitalypateshman/Study/Diplom/.venv/lib/python3.9/site-packages/django/db/models/sql/compiler.py", line 1562, in execute_sql
    cursor.execute(sql, params)
  File "/Users/vitalypateshman/Study/Diplom/.venv/lib/python3.9/site-packages/django/db/backends/utils.py", line 102, in execute
    return super().execute(sql, params)
  File "/Users/vitalypateshman/Study/Diplom/.venv/lib/python3.9/site-packages/django/db/backends/utils.py", line 67, in execute
    return self._execute_with_wrappers(
  File "/Users/vitalypateshman/Study/Diplom/.venv/lib/python3.9/site-packages/django/db/backends/utils.py", line 80, in _execute_with_wrappers
    return executor(sql, params, many, context)
  File "/Users/vitalypateshman/Study/Diplom/.venv/lib/python3.9/site-packages/django/db/backends/utils.py", line 89, in _execute
    return self.cursor.execute(sql, params)
  File "/Users/vitalypateshman/Study/Diplom/.venv/lib/python3.9/site-packages/django/db/utils.py", line 91, in __exit__
    raise dj_exc_value.with_traceback(traceback) from exc_value
  File "/Users/vitalypateshman/Study/Diplom/.venv/lib/python3.9/site-packages/django/db/backends/utils.py", line 89, in _execute
    return self.cursor.execute(sql, params)
django.db.utils.ProgrammingError: relation "accounts_customuser" does not exist
LINE 1: ...is_active", "accounts_customuser"."is_staff" FROM "accounts_...
                                                             ^

2024-08-14 06:59:00,772 ERROR django.server "POST /admin/login/?next=/admin/ HTTP/1.1" 500 224495
2024-08-14 07:00:31,980 INFO django.server "GET /admin/ HTTP/1.1" 302 0
2024-08-14 07:00:31,989 INFO django.server "GET /admin/login/?next=/admin/ HTTP/1.1" 200 4181
2024-08-14 07:03:44,381 ERROR django.request Internal Server Error: /admin/login/
Traceback (most recent call last):
  File "/Users/vitalypateshman/Study/Diplom/.venv/lib/python3.9/site-packages/django/db/backends/utils.py", line 89, in _execute
    return self.cursor.execute(sql, params)
psycopg2.errors.UndefinedTable: relation "accounts_customuser" does not exist
LINE 1: ...is_active", "accounts_customuser"."is_staff" FROM "accounts_...
                                                             ^


The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/Users/vitalypateshman/Study/Diplom/.venv/lib/python3.9/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/Users/vitalypateshman/Study/Diplom/.venv/lib/python3.9/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Users/vitalypateshman/Study/Diplom/.venv/lib/python3.9/site-packages/django/utils/decorators.py", line 46, in _wrapper
    return bound_method(*args, **kwargs)
  File "/Users/vitalypateshman/Study/Diplom/.venv/lib/python3.9/site-packages/django/views/decorators/cache.py", line 62, in _wrapper_view_func
    response = view_func(request, *args, **kwargs)
  File "/Users/vitalypateshman/Study/Diplom/.venv/lib/python3.9/site-packages/django/contrib/admin/sites.py", line 441, in login
    return LoginView.as_view(**defaults)(request)
  File "/Users/vitalypateshman/Study/Diplom/.venv/lib/python3.9/site-packages/django/views/generic/base.py", line 104, in view
    return self.dispatch(request, *args, **kwargs)
  File "/Users/vitalypateshman/Study/Diplom/.venv/lib/python3.9/site-packages/django/utils/decorators.py", line 46, in _wrapper
    return bound_method(*args, **kwargs)
  File "/Users/vitalypateshman/Study/Diplom/.venv/lib/python3.9/site-packages/django/views/decorators/debug.py", line 92, in sensitive_post_parameters_wrapper
    return view(request, *args, **kwargs)
  File "/Users/vitalypateshman/Study/Diplom/.venv/lib/python3.9/site-packages/django/utils/decorators.py", line 46, in _wrapper
    return bound_method(*args, **kwargs)
  File "/Users/vitalypateshman/Study/Diplom/.venv/lib/python3.9/site-packages/django/utils/decorators.py", line 134, in _wrapper_view
    response = view_func(request, *args, **kwargs)
  File "/Users/vitalypateshman/Study/Diplom/.venv/lib/python3.9/site-packages/django/utils/decorators.py", line 46, in _wrapper
    return bound_method(*args, **kwargs)
  File "/Users/vitalypateshman/Study/Diplom/.venv/lib/python3.9/site-packages/django/views/decorators/cache.py", line 62, in _wrapper_view_func
    response = view_func(request, *args, **kwargs)
  File "/Users/vitalypateshman/Study/Diplom/.venv/lib/python3.9/site-packages/django/contrib/auth/views.py", line 90, in dispatch
    return super().dispatch(request, *args, **kwargs)
  File "/Users/vitalypateshman/Study/Diplom/.venv/lib/python3.9/site-packages/django/views/generic/base.py", line 143, in dispatch
    return handler(request, *args, **kwargs)
  File "/Users/vitalypateshman/Study/Diplom/.venv/lib/python3.9/site-packages/django/views/generic/edit.py", line 152, in post
    if form.is_valid():
  File "/Users/vitalypateshman/Study/Diplom/.venv/lib/python3.9/site-packages/django/forms/forms.py", line 201, in is_valid
    return self.is_bound and not self.errors
  File "/Users/vitalypateshman/Study/Diplom/.venv/lib/python3.9/site-packages/django/forms/forms.py", line 196, in errors
    self.full_clean()
  File "/Users/vitalypateshman/Study/Diplom/.venv/lib/python3.9/site-packages/django/forms/forms.py", line 434, in full_clean
    self._clean_form()
  File "/Users/vitalypateshman/Study/Diplom/.venv/lib/python3.9/site-packages/django/forms/forms.py", line 455, in _clean_form
    cleaned_data = self.clean()
  File "/Users/vitalypateshman/Study/Diplom/.venv/lib/python3.9/site-packages/django/contrib/auth/forms.py", line 250, in clean
    self.user_cache = authenticate(
  File "/Users/vitalypateshman/Study/Diplom/.venv/lib/python3.9/site-packages/django/views/decorators/debug.py", line 42, in sensitive_variables_wrapper
    return func(*func_args, **func_kwargs)
  File "/Users/vitalypateshman/Study/Diplom/.venv/lib/python3.9/site-packages/django/contrib/auth/__init__.py", line 77, in authenticate
    user = backend.authenticate(request, **credentials)
  File "/Users/vitalypateshman/Study/Diplom/.venv/lib/python3.9/site-packages/django/contrib/auth/backends.py", line 46, in authenticate
    user = UserModel._default_manager.get_by_natural_key(username)
  File "/Users/vitalypateshman/Study/Diplom/.venv/lib/python3.9/site-packages/django/contrib/auth/base_user.py", line 54, in get_by_natural_key
    return self.get(**{self.model.USERNAME_FIELD: username})
  File "/Users/vitalypateshman/Study/Diplom/.venv/lib/python3.9/site-packages/django/db/models/manager.py", line 87, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
  File "/Users/vitalypateshman/Study/Diplom/.venv/lib/python3.9/site-packages/django/db/models/query.py", line 633, in get
    num = len(clone)
  File "/Users/vitalypateshman/Study/Diplom/.venv/lib/python3.9/site-packages/django/db/models/query.py", line 380, in __len__
    self._fetch_all()
  File "/Users/vitalypateshman/Study/Diplom/.venv/lib/python3.9/site-packages/django/db/models/query.py", line 1881, in _fetch_all
    self._result_cache = list(self._iterable_class(self))
  File "/Users/vitalypateshman/Study/Diplom/.venv/lib/python3.9/site-packages/django/db/models/query.py", line 91, in __iter__
    results = compiler.execute_sql(
  File "/Users/vitalypateshman/Study/Diplom/.venv/lib/python3.9/site-packages/django/db/models/sql/compiler.py", line 1562, in execute_sql
    cursor.execute(sql, params)
  File "/Users/vitalypateshman/Study/Diplom/.venv/lib/python3.9/site-packages/django/db/backends/utils.py", line 102, in execute
    return super().execute(sql, params)
  File "/Users/vitalypateshman/Study/Diplom/.venv/lib/python3.9/site-packages/django/db/backends/utils.py", line 67, in execute
    return self._execute_with_wrappers(
  File "/Users/vitalypateshman/Study/Diplom/.venv/lib/python3.9/site-packages/django/db/backends/utils.py", line 80, in _execute_with_wrappers
    return executor(sql, params, many, context)
  File "/Users/vitalypateshman/Study/Diplom/.venv/lib/python3.9/site-packages/django/db/backends/utils.py", line 89, in _execute
    return self.cursor.execute(sql, params)
  File "/Users/vitalypateshman/Study/Diplom/.venv/lib/python3.9/site-packages/django/db/utils.py", line 91, in __exit__
    raise dj_exc_value.with_traceback(traceback) from exc_value
  File "/Users/vitalypateshman/Study/Diplom/.venv/lib/python3.9/site-packages/django/db/backends/utils.py", line 89, in _execute
    return self.cursor.execute(sql, params)
django.db.utils.ProgrammingError: relation "accounts_customuser" does not exist
LINE 1: ...is_active", "accounts_customuser"."is_staff" FROM "accounts_...
                                                             ^

2024-08-14 07:03:44,382 ERROR django.server "POST /admin/login/?next=/admin/ HTTP/1.1" 500 224495
2024-08-14 08:55:44,418 INFO django.utils.autoreload /Users/vitalypateshman/Study/Diplom/backend/settings.py changed, reloading.
2024-08-14 08:55:44,699 INFO django.utils.autoreload Watching for file changes with StatReloader
2024-08-14 08:56:42,688 INFO django.utils.autoreload Watching for file changes with StatReloader
2024-08-14 08:57:06,317 WARNING django.request Not Found: /
2024-08-14 08:57:06,317 WARNING django.server "GET / HTTP/1.1" 404 3042
2024-08-14 08:57:14,697 INFO django.server "GET /admin/ HTTP/1.1" 302 0
2024-08-14 08:57:14,720 INFO django.server "GET /admin/login/?next=/admin/ HTTP/1.1" 200 4181
2024-08-14 08:57:25,539 INFO django.server "POST /admin/login/?next=/admin/ HTTP/1.1" 200 4345
2024-08-14 09:16:45,038 INFO django.utils.autoreload Watching for file changes with StatReloader
2024-08-14 09:16:54,532 INFO django.server "GET /admin/ HTTP/1.1" 302 0
2024-08-14 09:16:54,548 INFO django.server "GET /admin/login/?next=/admin/ HTTP/1.1" 200 4181
2024-08-14 09:16:54,558 INFO django.server "GET /static/admin/css/base.css HTTP/1.1" 304 0
2024-08-14 09:16:54,559 INFO django.server "GET /static/admin/css/nav_sidebar.css HTTP/1.1" 304 0
2024-08-14 09:16:54,559 INFO django.server "GET /static/admin/css/dark_mode.css HTTP/1.1" 304 0
2024-08-14 09:16:54,571 INFO django.server "GET /static/admin/css/login.css HTTP/1.1" 304 0
2024-08-14 09:16:54,572 INFO django.server "GET /static/admin/js/theme.js HTTP/1.1" 304 0
2024-08-14 09:16:54,572 INFO django.server "GET /static/admin/css/responsive.css HTTP/1.1" 304 0
2024-08-14 09:16:54,573 INFO django.server "GET /static/admin/js/nav_sidebar.js HTTP/1.1" 304 0
2024-08-14 09:17:01,545 INFO django.server "POST /admin/login/?next=/admin/ HTTP/1.1" 200 4345
2024-08-14 09:17:07,527 INFO django.server "POST /admin/login/?next=/admin/ HTTP/1.1" 200 4346
2024-08-14 09:17:24,168 INFO django.server "POST /admin/login/?next=/admin/ HTTP/1.1" 200 4345
2024-08-14 09:17:37,557 INFO django.server "POST /admin/login/?next=/admin/ HTTP/1.1" 200 4345
2024-08-14 09:17:42,286 INFO django.server "POST /admin/login/?next=/admin/ HTTP/1.1" 200 4346
2024-08-14 09:17:58,103 WARNING django.request Not Found: /
2024-08-14 09:17:58,103 WARNING django.server "GET / HTTP/1.1" 404 3042
2024-08-14 09:18:11,857 INFO django.server "GET /admin/ HTTP/1.1" 302 0
2024-08-14 09:18:11,865 INFO django.server "GET /admin/login/?next=/admin/ HTTP/1.1" 200 4181
2024-08-14 09:18:21,891 INFO django.server "POST /admin/login/?next=/admin/ HTTP/1.1" 200 4352
2024-08-14 09:19:20,286 INFO django.server "POST /admin/login/?next=/admin/ HTTP/1.1" 200 4345
2024-08-14 09:25:02,489 INFO django.server "POST /admin/login/?next=/admin/ HTTP/1.1" 200 4342
2024-08-14 09:25:11,017 INFO django.server "POST /admin/login/?next=/admin/ HTTP/1.1" 200 4345
2024-08-14 09:25:18,461 INFO django.server "POST /admin/login/?next=/admin/ HTTP/1.1" 200 4345
2024-08-14 09:25:44,209 INFO django.server "POST /admin/login/?next=/admin/ HTTP/1.1" 200 4345
2024-08-14 09:26:12,575 INFO django.utils.autoreload Watching for file changes with StatReloader
2024-08-14 09:26:31,495 INFO django.server "GET /admin/ HTTP/1.1" 302 0
2024-08-14 09:26:31,513 INFO django.server "GET /admin/login/?next=/admin/ HTTP/1.1" 200 4181
2024-08-14 09:26:43,091 INFO django.server "POST /admin/login/?next=/admin/ HTTP/1.1" 200 4346
2024-08-14 09:27:03,581 INFO django.server "POST /admin/login/?next=/admin/ HTTP/1.1" 200 4346
2024-08-14 09:27:07,752 INFO django.server "POST /admin/login/?next=/admin/ HTTP/1.1" 200 4345
2024-08-14 09:27:12,589 INFO django.server "POST /admin/login/?next=/admin/ HTTP/1.1" 200 4345
2024-08-14 09:27:23,802 INFO django.server "POST /admin/login/?next=/admin/ HTTP/1.1" 200 4346
2024-08-14 09:27:51,780 INFO django.utils.autoreload Watching for file changes with StatReloader
2024-08-14 09:27:57,013 INFO django.server "GET /admin/login/?next=/admin/ HTTP/1.1" 200 4181
2024-08-14 09:28:04,113 INFO django.server "POST /admin/login/?next=/admin/ HTTP/1.1" 200 4346
2024-08-14 09:30:07,332 INFO django.utils.autoreload Watching for file changes with StatReloader
2024-08-14 09:30:11,632 INFO django.server "GET /admin/login/?next=/admin/ HTTP/1.1" 200 4181
2024-08-14 09:30:17,856 INFO django.server "POST /admin/login/?next=/admin/ HTTP/1.1" 200 4345
2024-08-14 09:30:28,519 INFO django.server "POST /admin/login/?next=/admin/ HTTP/1.1" 200 4346
2024-08-14 09:30:34,502 INFO django.server "POST /admin/login/?next=/admin/ HTTP/1.1" 200 4347
2024-08-14 09:30:41,717 INFO django.server "POST /admin/login/?next=/admin/ HTTP/1.1" 200 4346
2024-08-14 09:31:13,287 INFO django.utils.autoreload Watching for file changes with StatReloader
2024-08-14 09:31:20,790 WARNING django.request Not Found: /
2024-08-14 09:31:20,790 WARNING django.server "GET / HTTP/1.1" 404 3042
2024-08-14 09:31:20,842 WARNING django.request Not Found: /favicon.ico
2024-08-14 09:31:20,842 WARNING django.server "GET /favicon.ico HTTP/1.1" 404 3093
2024-08-14 09:31:25,757 INFO django.server "GET /admin HTTP/1.1" 301 0
2024-08-14 09:31:25,768 INFO django.server "GET /admin/ HTTP/1.1" 302 0
2024-08-14 09:31:25,781 INFO django.server "GET /admin/login/?next=/admin/ HTTP/1.1" 200 4181
2024-08-14 09:31:25,798 INFO django.server "GET /static/admin/js/theme.js HTTP/1.1" 200 1943
2024-08-14 09:31:25,799 INFO django.server "GET /static/admin/js/nav_sidebar.js HTTP/1.1" 200 3063
2024-08-14 09:31:25,799 INFO django.server "GET /static/admin/css/login.css HTTP/1.1" 200 958
2024-08-14 09:31:25,799 INFO django.server "GET /static/admin/css/base.css HTTP/1.1" 200 21310
2024-08-14 09:31:25,799 INFO django.server "GET /static/admin/css/dark_mode.css HTTP/1.1" 200 2929
2024-08-14 09:31:25,799 INFO django.server "GET /static/admin/css/nav_sidebar.css HTTP/1.1" 200 2694
2024-08-14 09:31:25,800 INFO django.server "GET /static/admin/css/responsive.css HTTP/1.1" 200 18559
2024-08-14 09:31:33,611 INFO django.server "POST /admin/login/?next=/admin/ HTTP/1.1" 200 4345
2024-08-14 09:31:46,598 INFO django.server "POST /admin/login/?next=/admin/ HTTP/1.1" 200 4346
2024-08-14 09:31:54,168 INFO django.server "POST /admin/login/?next=/admin/ HTTP/1.1" 200 4345
2024-08-14 09:32:01,764 INFO django.server "POST /admin/login/?next=/admin/ HTTP/1.1" 200 4346
2024-08-14 09:32:07,257 INFO django.server "POST /admin/login/?next=/admin/ HTTP/1.1" 200 4346
2024-08-14 09:32:25,632 INFO django.server "POST /admin/login/?next=/admin/ HTTP/1.1" 200 4346
2024-08-14 09:39:00,016 INFO django.utils.autoreload Watching for file changes with StatReloader
2024-08-14 09:39:06,583 INFO django.server "GET /admin/login/?next=/admin/ HTTP/1.1" 200 4181
2024-08-14 09:39:15,698 INFO django.server "POST /admin/login/?next=/admin/ HTTP/1.1" 200 4346
2024-08-14 09:43:47,227 INFO django.server "GET /admin/login/?next=/admin/ HTTP/1.1" 200 4181
2024-08-14 09:43:56,766 INFO django.server "POST /admin/login/?next=/admin/ HTTP/1.1" 200 4346
2024-08-14 09:44:06,057 INFO django.server "GET /admin/ HTTP/1.1" 302 0
2024-08-14 09:44:06,066 INFO django.server "GET /admin/login/?next=/admin/ HTTP/1.1" 200 4181
2024-08-14 09:44:19,193 INFO django.server "POST /admin/login/?next=/admin/ HTTP/1.1" 200 4345
2024-08-14 09:47:47,944 INFO django.utils.autoreload Watching for file changes with StatReloader
2024-08-14 09:47:52,784 INFO django.server "GET /admin/login/?next=/admin/ HTTP/1.1" 200 4181
2024-08-14 09:48:00,874 INFO django.server "POST /admin/login/?next=/admin/ HTTP/1.1" 302 0
2024-08-14 09:48:00,891 INFO django.server "GET /admin/ HTTP/1.1" 200 6490
2024-08-14 09:48:00,905 INFO django.server "GET /static/admin/css/dashboard.css HTTP/1.1" 200 441
2024-08-14 09:48:00,914 INFO django.server "GET /static/admin/img/icon-addlink.svg HTTP/1.1" 200 331
2024-08-14 09:48:00,914 INFO django.server "GET /static/admin/img/icon-changelink.svg HTTP/1.1" 200 380
2024-08-14 09:48:18,406 INFO django.server "GET /admin/accounts/customuser/ HTTP/1.1" 200 10286
2024-08-14 09:48:18,426 INFO django.server "GET /static/admin/css/changelists.css HTTP/1.1" 200 6566
2024-08-14 09:48:18,430 INFO django.server "GET /static/admin/js/core.js HTTP/1.1" 200 5682
2024-08-14 09:48:18,430 INFO django.server "GET /static/admin/js/jquery.init.js HTTP/1.1" 200 347
2024-08-14 09:48:18,433 INFO django.server "GET /admin/jsi18n/ HTTP/1.1" 200 3343
2024-08-14 09:48:18,434 INFO django.server "GET /static/admin/js/admin/RelatedObjectLookups.js HTTP/1.1" 200 8943
2024-08-14 09:48:18,435 INFO django.server "GET /static/admin/js/prepopulate.js HTTP/1.1" 200 1531
2024-08-14 09:48:18,436 INFO django.server "GET /static/admin/js/actions.js HTTP/1.1" 200 7872
2024-08-14 09:48:18,436 INFO django.server "GET /static/admin/js/urlify.js HTTP/1.1" 200 7887
2024-08-14 09:48:18,438 INFO django.server "GET /static/admin/img/icon-yes.svg HTTP/1.1" 200 436
2024-08-14 09:48:18,439 INFO django.server "GET /static/admin/img/search.svg HTTP/1.1" 200 458
2024-08-14 09:48:18,441 INFO django.server "GET /static/admin/js/vendor/jquery/jquery.js HTTP/1.1" 200 292458
2024-08-14 09:48:18,442 INFO django.server "GET /static/admin/js/vendor/xregexp/xregexp.js HTTP/1.1" 200 232381
2024-08-14 09:48:18,443 INFO django.server "GET /static/admin/js/filters.js HTTP/1.1" 200 978
2024-08-14 09:48:18,453 INFO django.server "GET /static/admin/img/sorting-icons.svg HTTP/1.1" 200 1097
2024-08-14 09:48:18,453 INFO django.server "GET /static/admin/img/tooltag-add.svg HTTP/1.1" 200 331
2024-08-14 09:48:27,276 INFO django.server "GET /admin/accounts/customuser/add/ HTTP/1.1" 200 14844
2024-08-14 09:48:27,287 INFO django.server "GET /static/admin/js/prepopulate_init.js HTTP/1.1" 200 586
2024-08-14 09:48:27,289 INFO django.server "GET /static/admin/css/forms.css HTTP/1.1" 200 9090
2024-08-14 09:48:27,289 INFO django.server "GET /static/admin/js/inlines.js HTTP/1.1" 200 15526
2024-08-14 09:48:27,293 INFO django.server "GET /admin/jsi18n/ HTTP/1.1" 200 3343
2024-08-14 09:48:27,293 INFO django.server "GET /static/admin/css/widgets.css HTTP/1.1" 200 11921
2024-08-14 09:48:27,295 INFO django.server "GET /static/admin/js/change_form.js HTTP/1.1" 200 606
2024-08-14 09:54:47,982 INFO django.utils.autoreload Watching for file changes with StatReloader
2024-08-14 09:54:54,031 INFO django.server "GET /admin/ HTTP/1.1" 200 6490
2024-08-14 09:55:01,888 INFO django.server "GET /admin/authtoken/tokenproxy/add/ HTTP/1.1" 200 9479
2024-08-14 09:55:01,913 INFO django.server "GET /static/admin/img/icon-viewlink.svg HTTP/1.1" 200 581
2024-08-14 09:55:01,913 INFO django.server "GET /admin/jsi18n/ HTTP/1.1" 200 3343
2024-08-14 09:55:09,163 INFO django.server "GET /admin/accounts/customuser/add/ HTTP/1.1" 200 14844
2024-08-14 09:55:09,179 INFO django.server "GET /admin/jsi18n/ HTTP/1.1" 200 3343
2024-08-14 09:55:42,413 INFO django.server "POST /admin/accounts/customuser/add/ HTTP/1.1" 200 15006
2024-08-14 09:55:42,432 INFO django.server "GET /admin/jsi18n/ HTTP/1.1" 200 3343
2024-08-14 09:56:44,450 INFO django.server "POST /admin/accounts/customuser/add/ HTTP/1.1" 200 15006
2024-08-14 09:56:44,464 INFO django.server "GET /admin/jsi18n/ HTTP/1.1" 200 3343
2024-08-14 09:57:04,954 INFO django.server "POST /admin/accounts/customuser/add/ HTTP/1.1" 302 0
2024-08-14 09:57:04,972 INFO django.server "GET /admin/accounts/customuser/2/change/ HTTP/1.1" 200 14432
2024-08-14 09:57:04,990 INFO django.server "GET /admin/jsi18n/ HTTP/1.1" 200 3343
2024-08-14 09:57:11,354 INFO django.server "GET /admin/accounts/customuser/add/ HTTP/1.1" 200 14844
2024-08-14 09:57:11,374 INFO django.server "GET /admin/jsi18n/ HTTP/1.1" 200 3343
2024-08-14 09:57:26,420 INFO django.server "POST /admin/accounts/customuser/add/ HTTP/1.1" 302 0
2024-08-14 09:57:26,438 INFO django.server "GET /admin/accounts/customuser/3/change/ HTTP/1.1" 200 14440
2024-08-14 09:57:26,456 INFO django.server "GET /admin/jsi18n/ HTTP/1.1" 200 3343
2024-08-14 09:57:31,089 INFO django.server "GET /admin/accounts/customuser/ HTTP/1.1" 200 11259
2024-08-14 09:57:31,102 INFO django.server "GET /static/admin/img/icon-no.svg HTTP/1.1" 200 560
2024-08-14 09:57:31,106 INFO django.server "GET /admin/jsi18n/ HTTP/1.1" 200 3343
2024-08-14 09:57:39,713 INFO django.server "GET /admin/accounts/customuser/2/change/ HTTP/1.1" 200 14202
2024-08-14 09:57:39,736 INFO django.server "GET /admin/jsi18n/ HTTP/1.1" 200 3343
2024-08-14 09:57:42,807 INFO django.server "GET /admin/accounts/customuser/ HTTP/1.1" 200 11259
2024-08-14 09:57:42,826 INFO django.server "GET /admin/jsi18n/ HTTP/1.1" 200 3343
2024-08-14 09:57:44,348 INFO django.server "GET /admin/accounts/customuser/add/ HTTP/1.1" 200 14844
2024-08-14 09:57:44,369 INFO django.server "GET /admin/jsi18n/ HTTP/1.1" 200 3343
2024-08-14 09:58:00,111 INFO django.server "POST /admin/accounts/customuser/add/ HTTP/1.1" 302 0
2024-08-14 09:58:00,127 INFO django.server "GET /admin/accounts/customuser/4/change/ HTTP/1.1" 200 14432
2024-08-14 09:58:00,144 INFO django.server "GET /admin/jsi18n/ HTTP/1.1" 200 3343
2024-08-14 09:58:03,370 INFO django.server "GET /admin/authtoken/tokenproxy/ HTTP/1.1" 200 8105
2024-08-14 09:58:05,981 INFO django.server "GET /admin/authtoken/tokenproxy/add/ HTTP/1.1" 200 9587
2024-08-14 09:58:06,003 INFO django.server "GET /admin/jsi18n/ HTTP/1.1" 200 3343
2024-08-14 09:58:11,586 INFO django.server "GET /admin/accounts/customuser/add/?_to_field=id&_popup=1 HTTP/1.1" 200 10708
2024-08-14 09:58:11,598 INFO django.server "GET /admin/jsi18n/ HTTP/1.1" 200 3343
2024-08-14 09:58:25,652 INFO django.server "POST /admin/authtoken/tokenproxy/add/ HTTP/1.1" 302 0
2024-08-14 09:58:25,678 INFO django.server "GET /admin/authtoken/tokenproxy/ HTTP/1.1" 200 9336
2024-08-14 09:58:28,222 INFO django.server "GET /admin/authtoken/tokenproxy/add/ HTTP/1.1" 200 9587
2024-08-14 09:58:28,249 INFO django.server "GET /admin/jsi18n/ HTTP/1.1" 200 3343
2024-08-14 09:58:30,971 INFO django.server "POST /admin/authtoken/tokenproxy/add/ HTTP/1.1" 302 0
2024-08-14 09:58:30,999 INFO django.server "GET /admin/authtoken/tokenproxy/ HTTP/1.1" 200 9570
2024-08-14 09:58:32,824 INFO django.server "GET /admin/authtoken/tokenproxy/add/ HTTP/1.1" 200 9587
2024-08-14 09:58:32,841 INFO django.server "GET /admin/jsi18n/ HTTP/1.1" 200 3343
2024-08-14 09:58:36,047 INFO django.server "POST /admin/authtoken/tokenproxy/add/ HTTP/1.1" 302 0
2024-08-14 09:58:36,076 INFO django.server "GET /admin/authtoken/tokenproxy/ HTTP/1.1" 200 9803
2024-08-14 09:58:39,269 INFO django.server "GET /admin/server/document/ HTTP/1.1" 200 7723
2024-08-14 09:58:39,297 INFO django.server "GET /admin/jsi18n/ HTTP/1.1" 200 3343
2024-08-14 09:58:41,177 INFO django.server "GET /admin/server/document/add/ HTTP/1.1" 200 13757
2024-08-14 09:58:41,198 INFO django.server "GET /static/admin/js/calendar.js HTTP/1.1" 200 8466
2024-08-14 09:58:41,198 INFO django.server "GET /static/admin/js/admin/DateTimeShortcuts.js HTTP/1.1" 200 19319
2024-08-14 09:58:41,201 INFO django.server "GET /admin/jsi18n/ HTTP/1.1" 200 3343
2024-08-14 09:58:41,218 INFO django.server "GET /static/admin/img/icon-calendar.svg HTTP/1.1" 200 1086
2024-08-14 09:58:41,219 INFO django.server "GET /static/admin/img/icon-clock.svg HTTP/1.1" 200 677
2024-08-14 09:59:18,905 INFO django.server "POST /admin/server/document/add/ HTTP/1.1" 302 0
2024-08-14 09:59:18,931 INFO django.server "GET /admin/server/document/ HTTP/1.1" 200 10077
2024-08-14 09:59:18,950 INFO django.server "GET /admin/jsi18n/ HTTP/1.1" 200 3343
2024-08-14 09:59:29,129 WARNING django.request Not Found: /
2024-08-14 09:59:29,129 WARNING django.server "GET / HTTP/1.1" 404 3042
2024-08-14 09:59:44,089 INFO django.server "GET /login HTTP/1.1" 301 0
2024-08-14 09:59:44,124 WARNING django.request Method Not Allowed: /login/
2024-08-14 09:59:44,125 WARNING django.server "GET /login/ HTTP/1.1" 405 5601
2024-08-14 09:59:44,138 INFO django.server "GET /static/rest_framework/css/prettify.css HTTP/1.1" 304 0
2024-08-14 09:59:44,138 INFO django.server "GET /static/rest_framework/css/bootstrap.min.css HTTP/1.1" 304 0
2024-08-14 09:59:44,140 INFO django.server "GET /static/rest_framework/css/default.css HTTP/1.1" 304 0
2024-08-14 09:59:44,141 INFO django.server "GET /static/rest_framework/css/bootstrap-tweaks.css HTTP/1.1" 304 0
2024-08-14 09:59:44,143 INFO django.server "GET /static/rest_framework/js/jquery-3.7.1.min.js HTTP/1.1" 304 0
2024-08-14 09:59:44,143 INFO django.server "GET /static/rest_framework/js/bootstrap.min.js HTTP/1.1" 304 0
2024-08-14 09:59:44,143 INFO django.server "GET /static/rest_framework/js/ajax-form.js HTTP/1.1" 304 0
2024-08-14 09:59:44,144 INFO django.server "GET /static/rest_framework/js/csrf.js HTTP/1.1" 304 0
2024-08-14 09:59:44,144 INFO django.server "GET /static/rest_framework/js/prettify-min.js HTTP/1.1" 304 0
2024-08-14 09:59:44,145 INFO django.server "GET /static/rest_framework/js/default.js HTTP/1.1" 304 0
2024-08-14 09:59:44,145 INFO django.server "GET /static/rest_framework/js/load-ajax-form.js HTTP/1.1" 304 0
2024-08-14 09:59:44,146 INFO django.server "GET /static/rest_framework/img/grid.png HTTP/1.1" 304 0
2024-08-14 14:25:01,397 INFO django.utils.autoreload Watching for file changes with StatReloader
2024-08-14 14:25:07,588 INFO django.server "OPTIONS /login/ HTTP/1.1" 200 0
2024-08-14 14:25:07,810 ERROR django.request Internal Server Error: /login/
Traceback (most recent call last):
  File "/Users/vitalypateshman/Study/Diplom/accounts/views.py", line 51, in login_view
    token = Token.objects.get(user=authenticated_user)
  File "/Users/vitalypateshman/Study/Diplom/.venv/lib/python3.9/site-packages/django/db/models/manager.py", line 87, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
  File "/Users/vitalypateshman/Study/Diplom/.venv/lib/python3.9/site-packages/django/db/models/query.py", line 637, in get
    raise self.model.DoesNotExist(
rest_framework.authtoken.models.Token.DoesNotExist: Token matching query does not exist.

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/vitalypateshman/Study/Diplom/.venv/lib/python3.9/site-packages/django/db/backends/utils.py", line 89, in _execute
    return self.cursor.execute(sql, params)
psycopg2.errors.NotNullViolation: null value in column "user_id" of relation "authtoken_token" violates not-null constraint
DETAIL:  Failing row contains (11c7b156b4db914f18fd208d0fbbafdda03e61bc, 2024-08-14 14:25:07.778183+00, null).


The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/Users/vitalypateshman/Study/Diplom/.venv/lib/python3.9/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/Users/vitalypateshman/Study/Diplom/.venv/lib/python3.9/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Users/vitalypateshman/Study/Diplom/.venv/lib/python3.9/site-packages/django/views/decorators/csrf.py", line 56, in wrapper_view
    return view_func(*args, **kwargs)
  File "/Users/vitalypateshman/Study/Diplom/.venv/lib/python3.9/site-packages/django/views/generic/base.py", line 104, in view
    return self.dispatch(request, *args, **kwargs)
  File "/Users/vitalypateshman/Study/Diplom/.venv/lib/python3.9/site-packages/rest_framework/views.py", line 509, in dispatch
    response = self.handle_exception(exc)
  File "/Users/vitalypateshman/Study/Diplom/.venv/lib/python3.9/site-packages/rest_framework/views.py", line 469, in handle_exception
    self.raise_uncaught_exception(exc)
  File "/Users/vitalypateshman/Study/Diplom/.venv/lib/python3.9/site-packages/rest_framework/views.py", line 480, in raise_uncaught_exception
    raise exc
  File "/Users/vitalypateshman/Study/Diplom/.venv/lib/python3.9/site-packages/rest_framework/views.py", line 506, in dispatch
    response = handler(request, *args, **kwargs)
  File "/Users/vitalypateshman/Study/Diplom/.venv/lib/python3.9/site-packages/rest_framework/decorators.py", line 50, in handler
    return func(*args, **kwargs)
  File "/Users/vitalypateshman/Study/Diplom/accounts/views.py", line 53, in login_view
    token = Token.objects.create(user=authenticated_user)
  File "/Users/vitalypateshman/Study/Diplom/.venv/lib/python3.9/site-packages/django/db/models/manager.py", line 87, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
  File "/Users/vitalypateshman/Study/Diplom/.venv/lib/python3.9/site-packages/django/db/models/query.py", line 658, in create
    obj.save(force_insert=True, using=self.db)
  File "/Users/vitalypateshman/Study/Diplom/.venv/lib/python3.9/site-packages/rest_framework/authtoken/models.py", line 33, in save
    return super().save(*args, **kwargs)
  File "/Users/vitalypateshman/Study/Diplom/.venv/lib/python3.9/site-packages/django/db/models/base.py", line 814, in save
    self.save_base(
  File "/Users/vitalypateshman/Study/Diplom/.venv/lib/python3.9/site-packages/django/db/models/base.py", line 877, in save_base
    updated = self._save_table(
  File "/Users/vitalypateshman/Study/Diplom/.venv/lib/python3.9/site-packages/django/db/models/base.py", line 1020, in _save_table
    results = self._do_insert(
  File "/Users/vitalypateshman/Study/Diplom/.venv/lib/python3.9/site-packages/django/db/models/base.py", line 1061, in _do_insert
    return manager._insert(
  File "/Users/vitalypateshman/Study/Diplom/.venv/lib/python3.9/site-packages/django/db/models/manager.py", line 87, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
  File "/Users/vitalypateshman/Study/Diplom/.venv/lib/python3.9/site-packages/django/db/models/query.py", line 1805, in _insert
    return query.get_compiler(using=using).execute_sql(returning_fields)
  File "/Users/vitalypateshman/Study/Diplom/.venv/lib/python3.9/site-packages/django/db/models/sql/compiler.py", line 1822, in execute_sql
    cursor.execute(sql, params)
  File "/Users/vitalypateshman/Study/Diplom/.venv/lib/python3.9/site-packages/django/db/backends/utils.py", line 102, in execute
    return super().execute(sql, params)
  File "/Users/vitalypateshman/Study/Diplom/.venv/lib/python3.9/site-packages/django/db/backends/utils.py", line 67, in execute
    return self._execute_with_wrappers(
  File "/Users/vitalypateshman/Study/Diplom/.venv/lib/python3.9/site-packages/django/db/backends/utils.py", line 80, in _execute_with_wrappers
    return executor(sql, params, many, context)
  File "/Users/vitalypateshman/Study/Diplom/.venv/lib/python3.9/site-packages/django/db/backends/utils.py", line 89, in _execute
    return self.cursor.execute(sql, params)
  File "/Users/vitalypateshman/Study/Diplom/.venv/lib/python3.9/site-packages/django/db/utils.py", line 91, in __exit__
    raise dj_exc_value.with_traceback(traceback) from exc_value
  File "/Users/vitalypateshman/Study/Diplom/.venv/lib/python3.9/site-packages/django/db/backends/utils.py", line 89, in _execute
    return self.cursor.execute(sql, params)
django.db.utils.IntegrityError: null value in column "user_id" of relation "authtoken_token" violates not-null constraint
DETAIL:  Failing row contains (11c7b156b4db914f18fd208d0fbbafdda03e61bc, 2024-08-14 14:25:07.778183+00, null).

2024-08-14 14:25:07,817 ERROR django.server "POST /login/ HTTP/1.1" 500 188976
2024-08-14 14:25:08,855 ERROR django.request Internal Server Error: /login/
Traceback (most recent call last):
  File "/Users/vitalypateshman/Study/Diplom/accounts/views.py", line 51, in login_view
    token = Token.objects.get(user=authenticated_user)
  File "/Users/vitalypateshman/Study/Diplom/.venv/lib/python3.9/site-packages/django/db/models/manager.py", line 87, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
  File "/Users/vitalypateshman/Study/Diplom/.venv/lib/python3.9/site-packages/django/db/models/query.py", line 637, in get
    raise self.model.DoesNotExist(
rest_framework.authtoken.models.Token.DoesNotExist: Token matching query does not exist.

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/vitalypateshman/Study/Diplom/.venv/lib/python3.9/site-packages/django/db/backends/utils.py", line 89, in _execute
    return self.cursor.execute(sql, params)
psycopg2.errors.NotNullViolation: null value in column "user_id" of relation "authtoken_token" violates not-null constraint
DETAIL:  Failing row contains (f46c9271222abc7b15c09bcd5ab7489150ca11c0, 2024-08-14 14:25:08.837498+00, null).


The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/Users/vitalypateshman/Study/Diplom/.venv/lib/python3.9/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/Users/vitalypateshman/Study/Diplom/.venv/lib/python3.9/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Users/vitalypateshman/Study/Diplom/.venv/lib/python3.9/site-packages/django/views/decorators/csrf.py", line 56, in wrapper_view
    return view_func(*args, **kwargs)
  File "/Users/vitalypateshman/Study/Diplom/.venv/lib/python3.9/site-packages/django/views/generic/base.py", line 104, in view
    return self.dispatch(request, *args, **kwargs)
  File "/Users/vitalypateshman/Study/Diplom/.venv/lib/python3.9/site-packages/rest_framework/views.py", line 509, in dispatch
    response = self.handle_exception(exc)
  File "/Users/vitalypateshman/Study/Diplom/.venv/lib/python3.9/site-packages/rest_framework/views.py", line 469, in handle_exception
    self.raise_uncaught_exception(exc)
  File "/Users/vitalypateshman/Study/Diplom/.venv/lib/python3.9/site-packages/rest_framework/views.py", line 480, in raise_uncaught_exception
    raise exc
  File "/Users/vitalypateshman/Study/Diplom/.venv/lib/python3.9/site-packages/rest_framework/views.py", line 506, in dispatch
    response = handler(request, *args, **kwargs)
  File "/Users/vitalypateshman/Study/Diplom/.venv/lib/python3.9/site-packages/rest_framework/decorators.py", line 50, in handler
    return func(*args, **kwargs)
  File "/Users/vitalypateshman/Study/Diplom/accounts/views.py", line 53, in login_view
    token = Token.objects.create(user=authenticated_user)
  File "/Users/vitalypateshman/Study/Diplom/.venv/lib/python3.9/site-packages/django/db/models/manager.py", line 87, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
  File "/Users/vitalypateshman/Study/Diplom/.venv/lib/python3.9/site-packages/django/db/models/query.py", line 658, in create
    obj.save(force_insert=True, using=self.db)
  File "/Users/vitalypateshman/Study/Diplom/.venv/lib/python3.9/site-packages/rest_framework/authtoken/models.py", line 33, in save
    return super().save(*args, **kwargs)
  File "/Users/vitalypateshman/Study/Diplom/.venv/lib/python3.9/site-packages/django/db/models/base.py", line 814, in save
    self.save_base(
  File "/Users/vitalypateshman/Study/Diplom/.venv/lib/python3.9/site-packages/django/db/models/base.py", line 877, in save_base
    updated = self._save_table(
  File "/Users/vitalypateshman/Study/Diplom/.venv/lib/python3.9/site-packages/django/db/models/base.py", line 1020, in _save_table
    results = self._do_insert(
  File "/Users/vitalypateshman/Study/Diplom/.venv/lib/python3.9/site-packages/django/db/models/base.py", line 1061, in _do_insert
    return manager._insert(
  File "/Users/vitalypateshman/Study/Diplom/.venv/lib/python3.9/site-packages/django/db/models/manager.py", line 87, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
  File "/Users/vitalypateshman/Study/Diplom/.venv/lib/python3.9/site-packages/django/db/models/query.py", line 1805, in _insert
    return query.get_compiler(using=using).execute_sql(returning_fields)
  File "/Users/vitalypateshman/Study/Diplom/.venv/lib/python3.9/site-packages/django/db/models/sql/compiler.py", line 1822, in execute_sql
    cursor.execute(sql, params)
  File "/Users/vitalypateshman/Study/Diplom/.venv/lib/python3.9/site-packages/django/db/backends/utils.py", line 102, in execute
    return super().execute(sql, params)
  File "/Users/vitalypateshman/Study/Diplom/.venv/lib/python3.9/site-packages/django/db/backends/utils.py", line 67, in execute
    return self._execute_with_wrappers(
  File "/Users/vitalypateshman/Study/Diplom/.venv/lib/python3.9/site-packages/django/db/backends/utils.py", line 80, in _execute_with_wrappers
    return executor(sql, params, many, context)
  File "/Users/vitalypateshman/Study/Diplom/.venv/lib/python3.9/site-packages/django/db/backends/utils.py", line 89, in _execute
    return self.cursor.execute(sql, params)
  File "/Users/vitalypateshman/Study/Diplom/.venv/lib/python3.9/site-packages/django/db/utils.py", line 91, in __exit__
    raise dj_exc_value.with_traceback(traceback) from exc_value
  File "/Users/vitalypateshman/Study/Diplom/.venv/lib/python3.9/site-packages/django/db/backends/utils.py", line 89, in _execute
    return self.cursor.execute(sql, params)
django.db.utils.IntegrityError: null value in column "user_id" of relation "authtoken_token" violates not-null constraint
DETAIL:  Failing row contains (f46c9271222abc7b15c09bcd5ab7489150ca11c0, 2024-08-14 14:25:08.837498+00, null).

2024-08-14 14:25:08,855 ERROR django.server "POST /login/ HTTP/1.1" 500 188976
2024-08-14 14:26:22,417 INFO django.server "POST /login/ HTTP/1.1" 200 135
2024-08-14 14:26:22,439 INFO django.server "OPTIONS /api/users/2/ HTTP/1.1" 200 0
2024-08-14 14:26:22,440 INFO django.server "OPTIONS /api/files/ HTTP/1.1" 200 0
2024-08-14 14:26:22,440 INFO django.server "OPTIONS /api/users/ HTTP/1.1" 200 0
2024-08-14 14:26:22,440 INFO django.server "OPTIONS /api/users/2/ HTTP/1.1" 200 0
2024-08-14 14:26:22,441 INFO django.server "OPTIONS /api/files/ HTTP/1.1" 200 0
2024-08-14 14:26:22,442 INFO django.server "OPTIONS /api/users/ HTTP/1.1" 200 0
2024-08-14 14:26:22,447 INFO django.server "GET /api/users/2/ HTTP/1.1" 200 42
2024-08-14 14:26:22,449 INFO django.server "GET /api/files/ HTTP/1.1" 200 2
2024-08-14 14:26:22,449 INFO django.server "GET /api/users/ HTTP/1.1" 200 44
2024-08-14 14:26:22,459 INFO django.server "GET /api/users/2/ HTTP/1.1" 200 42
2024-08-14 14:26:22,459 INFO django.server "GET /api/files/ HTTP/1.1" 200 2
2024-08-14 14:26:22,460 INFO django.server "GET /api/users/ HTTP/1.1" 200 44
2024-08-14 14:28:58,368 INFO django.server "POST /api/files/ HTTP/1.1" 201 306
2024-08-14 14:28:58,387 INFO django.server "GET /api/files/ HTTP/1.1" 200 308
2024-08-14 14:29:49,296 INFO django.server "POST /api/files/ HTTP/1.1" 201 321
2024-08-14 14:29:49,313 INFO django.server "GET /api/files/ HTTP/1.1" 200 630
2024-08-14 14:30:11,232 INFO django.server "OPTIONS /api/files/2 HTTP/1.1" 200 0
2024-08-14 14:30:11,244 INFO django.server "GET /api/files/2 HTTP/1.1" 301 0
2024-08-14 14:30:11,246 INFO django.server "OPTIONS /api/files/2/ HTTP/1.1" 200 0
2024-08-14 14:30:11,261 INFO django.server "GET /api/files/2/ HTTP/1.1" 200 321
2024-08-14 14:30:11,268 INFO django.server "GET /files/user1/%D1%84%D0%BE%D1%82%D0%BE-2.webp HTTP/1.1" 200 21572
2024-08-14 14:38:20,691 INFO django.server "OPTIONS /logout/ HTTP/1.1" 200 0
2024-08-14 14:38:20,704 INFO django.server "GET /logout/ HTTP/1.1" 200 65
2024-08-14 14:38:35,258 INFO django.server "POST /login/ HTTP/1.1" 200 135
2024-08-14 14:38:35,282 INFO django.server "OPTIONS /api/users/4/ HTTP/1.1" 200 0
2024-08-14 14:38:35,283 INFO django.server "OPTIONS /api/users/4/ HTTP/1.1" 200 0
2024-08-14 14:38:35,291 INFO django.server "GET /api/files/ HTTP/1.1" 200 2
2024-08-14 14:38:35,292 INFO django.server "GET /api/users/ HTTP/1.1" 200 44
2024-08-14 14:38:35,293 INFO django.server "GET /api/users/4/ HTTP/1.1" 200 42
2024-08-14 14:38:35,298 INFO django.server "GET /api/files/ HTTP/1.1" 200 2
2024-08-14 14:38:35,298 INFO django.server "GET /api/users/ HTTP/1.1" 200 44
2024-08-14 14:38:35,301 INFO django.server "GET /api/users/4/ HTTP/1.1" 200 42
2024-08-14 14:38:42,301 INFO django.server "GET /logout/ HTTP/1.1" 200 65
2024-08-14 14:38:52,759 INFO django.server "POST /login/ HTTP/1.1" 200 135
2024-08-14 14:38:52,782 INFO django.server "OPTIONS /api/users/3/ HTTP/1.1" 200 0
2024-08-14 14:38:52,782 INFO django.server "OPTIONS /api/users/3/ HTTP/1.1" 200 0
2024-08-14 14:38:52,791 INFO django.server "GET /api/users/3/ HTTP/1.1" 200 117
2024-08-14 14:38:52,791 INFO django.server "GET /api/files/ HTTP/1.1" 200 630
2024-08-14 14:38:52,791 INFO django.server "GET /api/users/ HTTP/1.1" 200 513
2024-08-14 14:38:52,802 INFO django.server "GET /api/users/3/ HTTP/1.1" 200 117
2024-08-14 14:38:52,803 INFO django.server "GET /api/files/ HTTP/1.1" 200 630
2024-08-14 14:38:52,803 INFO django.server "GET /api/users/ HTTP/1.1" 200 513
2024-08-14 14:39:02,658 INFO django.server "GET /api/users/3/ HTTP/1.1" 200 117
2024-08-14 14:39:02,669 INFO django.server "GET /api/users/3/ HTTP/1.1" 200 117
2024-08-14 14:39:02,681 INFO django.server "GET /api/users/3/ HTTP/1.1" 200 117
2024-08-14 14:39:02,692 INFO django.server "GET /api/users/3/ HTTP/1.1" 200 117
2024-08-14 14:39:02,693 INFO django.server "GET /api/users/ HTTP/1.1" 200 513
2024-08-14 14:39:02,704 INFO django.server "GET /api/users/ HTTP/1.1" 200 513
2024-08-14 14:39:16,342 INFO django.server "GET /api/users/ HTTP/1.1" 200 513
2024-08-14 14:39:16,349 INFO django.server "GET /api/files/ HTTP/1.1" 200 630
2024-08-14 14:39:16,357 INFO django.server "GET /api/users/ HTTP/1.1" 200 513
2024-08-14 14:39:16,359 INFO django.server "GET /api/files/ HTTP/1.1" 200 630
